src/screens/Accounts/Accounts.js

```diff
@@ -69,11 +76,12 @@ type Props = {
   accounts: Accounts,
   resetIncorrectPassword: Function,
   switchAccount: Function,
+  smartWalletState: Object,
```

Comment:
Would this be `reducers/smartWalletReducer#WalletReducerState`? If so, we could explicitly type it accordingly.